judgels-backends/sandalphon/sandalphon-client/src/test/java/judgels/sandalphon/submission/bundle/MultipleChoiceItemSubmissionGraderTests.java

```diff
@@ -0,0 +1,92 @@
+package judgels.sandalphon.submission.bundle;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import judgels.gabriel.api.Verdicts;
+import judgels.sandalphon.api.problem.bundle.Item;
+import judgels.sandalphon.api.problem.bundle.ItemType;
+import judgels.sandalphon.api.submission.bundle.ItemSubmission.Grading;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class MultipleChoiceItemSubmissionGraderTests {
+    private ObjectMapper mapper;
+    private Item testItem1;
+    private Item testItem2;
+
+    @BeforeEach
+    void before() {
+        mapper = new ObjectMapper();
+
+        testItem1 = new Item.Builder()
+                .jid("item1jid")
+                .meta("1")
+                .type(ItemType.MULTIPLE_CHOICE)
+                .config("{\"statement\":\"ini soal 1\",\"score\":4.0,\"penalty\":1.0,\""
+                        + "choices\":[{\"alias\":\"a\",\"content\":\"jawaban a\",\"isCorrect\":false},"
+                        + "{\"alias\":\"b\",\"content\":\"jawaban b (benar)\",\"isCorrect\":true},"
+                        + "{\"alias\":\"c\",\"content\":\"jawaban c\",\"isCorrect\":false},"
+                        + "{\"alias\":\"d\",\"content\":\"jawaban d\",\"isCorrect\":false},"
+                        + "{\"alias\":\"e\",\"content\":\"jawaban e\",\"isCorrect\":false}]}")
+                .build();
+
+        testItem2 = new Item.Builder()
+                .jid("item2jid")
+                .meta("2")
+                .type(ItemType.MULTIPLE_CHOICE)
+                .config("Invalid JSON")
+                .build();
+    }
+
+    @Test
+    void accepted() {
+        MultipleChoiceItemSubmissionGrader grader = new MultipleChoiceItemSubmissionGrader(mapper);
+
+        Grading grading = grader.grade(testItem1, "b");
+        assertThat(grading.getVerdict().getCode()).isEqualTo(Verdicts.ACCEPTED.getCode());
```

Comment:
```suggestion
        assertThat(grading.getVerdict()).isEqualTo(Verdicts.ACCEPTED);
```

(and all similar occurrences)

Reply:
OK, fixed